infra/cifuzz/filestore/git/git_test.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for git."""
+
+import filecmp
+import os
+import sys
+import tempfile
+import subprocess
+import unittest
+from unittest import mock
+
+# pylint: disable=wrong-import-position
+INFRA_DIR = os.path.dirname(
```

Comment:
I'm not sure you need this. It's mainly for importing files from the infra/ dir. Imports from the cifuzz dir should generally work (maybe this won't because it's in a subdir of cifuzz.


I have a branch that tries to solve this problem by moving the entrypoint files into infra/.
https://github.com/google/oss-fuzz/tree/cifuzz-directory-layout
But it doesn't seem to work perfectly. I think it breaks when I selectively run tests or something. I wish I knew how python's importing worked better.

Reply:
This was needed after all. Otherwise CI tests failed when we try to "import retry".

Reply:
> This was needed after all. Otherwise CI tests failed when we try to "import retry".

Right that makes sense, because retry.py is in infra/, not in infra/cifuzz